benchmarks/benchmark/tools/latency-profile/manifest-templates/service-account.yaml.tpl

```diff
@@ -0,0 +1,7 @@
+apiVersion: v1
+kind: ServiceAccount
```

Comment:
The infra/ creates a service account for the benchmarking tooling: https://github.com/GoogleCloudPlatform/ai-on-gke/blob/1f4c96861ad166fd8c2c7c1847e45c8b187e362c/benchmarks/infra/stage-2/main.tf#L31

Long-term, I think we'll want to clean up the service account story for both pathways:
1. people who already created service account via infra/ automation
2. people who need to create a new service account since they didn't use infra/ automation (aka BYOCluster)

Reply:
This change was diagnostic, has since been removed